apps/site/src/app/apply/sections/Components/SimpleRadio.tsx

```diff
@@ -0,0 +1,62 @@
+import RequiredAsterisk from "./RequiredAsterisk";
+
+interface TextfieldProps {
+	name: string;
+	values: Array<{
+		id: string;
+		labelText: string;
+		inputValue: string;
+	}>;
+	title: string;
+	titleClass: string;
+	containerClassTotal: string;
+	containerClassInputLabels: string;
+	containerClassValues: string;
+	labelClass: string;
+	isRequired: boolean;
+}
+
+export default function SimpleRadio({
+	name,
+	values,
+	title,
+	titleClass,
+	containerClassTotal,
+	containerClassInputLabels,
+	containerClassValues,
+	labelClass,
+	isRequired,
+}: TextfieldProps) {
+	return (
+		<div className={containerClassTotal}>
+			<p className={titleClass}>
+				{`${title} `}
+				{isRequired ? <RequiredAsterisk /> : <></>}
```

Comment:
I looked through the docs and it seems that ?? only works with null / undefined, which is a bit problematic because isRequired is a boolean, so I'm not entirely sure how I would implement this

Reply:
My mistake, I wasn't thinking straight. Sure, that works.